crates/burn-jit/src/codegen/dialect/gpu/shader.rs

```diff
@@ -21,6 +21,7 @@ pub enum Visibility {
 #[allow(missing_docs)]
 pub enum Elem {
     Float,
+    Half,
```

Comment:
There is no need to add `Half` here, `Float` should cover all float types of all precisions in this context.